app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -580,26 +591,19 @@ class LuciBuildService {
   }
 
   /// Creates postsubmit check runs for prod targets in supported repositories.
-  Future<void> createPostsubmitCheckRun(
+  Future<github.CheckRun?> createPostsubmitCheckRun(
     Commit commit,
     Target target,
-    Map<String, dynamic> rawUserData,
   ) async {
     if (!config.githubPostsubmitSupportedRepo(commit.slug) || target.value.bringup) {
-      return;
+      return null;
     }
-    final github.CheckRun checkRun = await githubChecksUtil.createCheckRun(
+    return await githubChecksUtil.createCheckRun(
```

Comment:
Do we need to recreate a check run or can we rerequest an existing checkrun: https://g3doc.corp.google.com/company/teams/eip-cloud/overground/user-guide/quickstart.md?cl=head#onboard-project

Reply:
Though I am not sure how it associates the new rerequested check run with the particular pr.